README.md

```diff
@@ -10,10 +10,14 @@ This repository was first announced on Elastic's blog post, [Elastic Security op
 
 
 ## Table of Contents
-- [Overview of this repository](#overview-of-this-repository)
-- [Getting started](#getting-started)
-- [How to contribute](#how-to-contribute)
-- [Licensing](#licensing)
+- [Detection Rules](#detection-rules)
```

Comment:
this seems redundant to me, so I think we could remove this one and unindent the others

Reply:
Honestly I don't know how these lines made it into the file. Removing them
[Table of Contents](#table-of-contents)
[Detection Rules](#detection-rules)